proposals/lip-0049.md

```diff
@@ -273,6 +273,9 @@ def apply(trs: Transaction, ccm: CCM) -> None:
     else:
         crossChainCommand = cross-chain command associated with (ccm.module, ccm.crossChainCommand)
         try:  
+            paramsSchema = parameter schema associated with (ccm.module, ccm.crossChainCommand)
```

Comment:
Minor comment: Perhaps we can add comments here (before line 276 and 279) to separate the steps of validating params and verifying the command, to make it easier to parse.